src/Countries/US/Georgia/GeorgiaIncome/V20180101/GeorgiaIncome.php

```diff
@@ -45,11 +45,11 @@ class GeorgiaIncome extends BaseGeorgiaIncome
     ];
 
     const STANDARD_DEDUCTIONS = [
-        self::FILING_SINGLE => 2300,
-        self::FILING_HEAD_OF_HOUSEHOLD => 2300,
-        self::FILING_MARRIED_SEPARATE => 1500,
-        self::FILING_MARRIED_JOINT_BOTH_WORKING => 1500,
-        self::FILING_MARRIED_JOINT_ONE_WORKING => 3000,
+        self::FILING_SINGLE => 4600,
+        self::FILING_HEAD_OF_HOUSEHOLD => 4600,
+        self::FILING_MARRIED_SEPARATE => 3000,
+        self::FILING_MARRIED_JOINT_BOTH_WORKING => 3000,
```

Comment:
should both JOINT be $6000? that's how i read the article but im not really not sure.

Reply:
I _think_ this is right, but man is it hard to tell.